Source/diablo.cpp

```diff
@@ -2078,7 +2078,7 @@ static void game_logic()
 		CheckCursMove();
 		track_process();
 	}
-	if (gbProcessPlayers) {
+	if (gbProcessPlayers) { //gbProcessPlayers is only set to false when Diablo dies, otherwise it's true

```

Comment:
This should be moved to where `gbProcessPlayers` is defined.

Reply:
Done.